.babelrc.json

```diff
@@ -5,9 +5,9 @@
       "@babel/preset-env",
       {
         "targets": {
-          "chrome": 100,
-          "safari": 15,
-          "firefox": 91
+          "chrome": 130,
+          "safari": 16,
+          "firefox": 115
         }
```

Comment:
Do we actually need this? I thought it would automatically read from the `browserslist` entry in `package.json`? https://babeljs.io/docs/babel-preset-env#browserslist-integration